packages/vscode-extension/package.json

```diff
@@ -716,7 +716,8 @@
     "test-compile": "tsc -p ./",
     "test-watch": "npm run copy-test-files && tsc -watch -p ./",
     "pretest": "npm run lint && npm run check-format && npm run test-compile",
-    "test:unit": "rimraf coverage && npm run compile && node out/test/unit/runTest.js",
+    "test:unit": "rimraf coverage && npm run compile && node out/test/unit/runTest.js && npm run test:unit:migration",
```

Comment:
Any change in `runTest.js` to exclude `migrate.test.ts` ?

Reply:
Removed migration test from `runTest.js`.